addon/active-model-adapter.js

```diff
@@ -148,7 +148,7 @@ const ActiveModelAdapter = RESTAdapter.extend({
     @param  {Object} payload
     @return {Object | DS.AdapterError} response
   */
-  handleResponse: function(status, headers, payload) {
+  handleResponse: function(status, headers, payload, requestData) {
```

Comment:
`requestData` isn't used within the method, so I wonder if we should update `.jshintrc`'s setting for `unused` to change it from `"vars"` to `true` so JSHint errors for unused arguments too...